### ISSUE TITLE
[BUG] Autothrottle not engaging

### ISSUE THREAD

**Issue by Sierrapappayankee** [flybywiresim/a32nx#3787]
**Mod Version**

cbd6c53
Custom FBW from installer. 

**Describe the bug**
The last two flights I've taken the autothrottle does not work at all on takeoff when I move it to the climb detent. I try re-engaging it but it doesn't work. Repeated attempts to turn it on don't work until I reach cruise level, then it goes to 'mach' setting. I then thought it was ok but it turned off again and aircraft over sped. I have to keep re-engaging the autothrottle 

**To Reproduce**

1. EGLL-EDDF, night flight, 02 March. climb to Cruise FL390. 
2. throttle up on takeoff and reduce to climb detent on demand
3.

**Expected behavior**
Autothust to engage when I select the climb detent as always

**Actual behavior**
explained above. did not engage until cruise

**References**

**Additional context**

I tried to assign different axes on my controllers to see if they where the problem but it still turns off

Was this working before/when did the issue start occurring?
only appeared since beginning of March. I update the custom fbw every day

Is this a problem in the vanilla unmodded game?
unmodded

Discord username (if different from GitHub):

**Comment by Benjozork** [comment]
Did you read the [throttle configuration guide for the experimental version](https://github.com/flybywiresim/a32nx/tree/fbw/docs) ?

**Comment by Sierrapappayankee** [comment]
> Did you read the [throttle configuration guide for the experimental version](https://github.com/flybywiresim/a32nx/tree/fbw/docs) ?

I haven't changed anything on my throttle axis? Surely I shouldn't have to experiment with dead zones?

**Comment by aguther** [comment]
Did you have FD disabled? Reports have reached me that if FD is off default ATHR seems to have issues.

**Comment by Sierrapappayankee** [comment]
> Did you have FD disabled? Reports have reached me that if FD is off default ATHR seems to have issues.

No, I never have the FD disabled. 
thanks for reply. :)

**Comment by Sierrapappayankee** [comment]
> Did you read the [throttle configuration guide for the experimental version](https://github.com/flybywiresim/a32nx/tree/fbw/docs) ?

So if this is related to the Throttle configuration being reset. How do I go about fixing it? The guide is a bit ambiguous.

**Comment by aguther** [comment]
Where do you have issues understanding?

**Comment by natdog021** [comment]
Question you mentioned climb detent are you going past climb on takeoff? Because if you don’t reach FLX/MCT or TOGA authrust won’t engage.

**Comment by Sierrapappayankee** [comment]
> Question you mentioned climb detent are you going past climb on takeoff? Because if you don’t reach FLX/MCT or TOGA authrust won’t engage.

I go to TOGA on takeoff then reduce to climb detent when required. 
Ive done quite a lot of testing now & autothrottle  even seems to sometimes turn off during FLX/MCT or TOGA.

**Comment by Sierrapappayankee** [comment]
> Where do you have issues understanding?

Am I supposed to try change the values listed until the problem goes away? 
why is this all of a sudden happening though? 
I tried re-installing. I also then Tried the stable & development build, all where the same: Autothrottle disengages a few seconds after going into climb detent. 
I tried changing the values as per the guide many times & that made no difference. 

It’s either a bug, or an issue with the throttle quadrant, which wouldn’t surprise me as Thrustmaster is rubbish quality!

**Comment by aguther** [comment]
Ok, that‘s a new information. If it happens also with stable and development, it‘s at least not related to custom fbw system. Have you checked if there is any assignment of the ATHR disconnect button that you‘re not aware of?

**Comment by Sierrapappayankee** [comment]
> Ok, that‘s a new information. If it happens also with stable and development, it‘s at least not related to custom fbw system. Have you checked if there is any assignment of the ATHR disconnect button that you‘re not aware of?

Thank you for suggesting this. I checked all assignments related to Autothrottle or the Throttles in general on my Joystick, Throttle, Keyboard & even mouse. It turns out there was an autothrottle disengage assigned to a side button on my mouse. 
I removed **everything** assigned & the problem seems to have gone away. 
This is strange because I have watched the Autothrottle turning off without me touching **anything**. This leaves me wondering if there is a faulty signal being sent! 
Thanks for your help & suggestions, I was very lonely without my beloved FBW A320!
